docs/_basic/listening_responding_shortcuts.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Listening and responding to shortcuts
+lang: en
+slug: shortcuts
+order: 8
+---
+
+<div class="section-content">
+
+The `shortcut()` method supports both [global shortcuts](https://api.slack.com/interactivity/shortcuts/using#global_shortcuts) and [message shortcuts](https://api.slack.com/interactivity/shortcuts/using#message_shortcuts).
+
+Shortcuts are invokable entry points to apps. Global shortcuts are available from within search in Slack. Message shortcuts are available in the context menus of messages. Your app can use the `shortcut()` method to listen to incoming shortcut events. The method requires a `callback_id` parameter of type `str` or `RegExp`.
+
+Shortcuts must be acknowledged with `ack()` to inform Slack that your app has received the event.
+
+Shortcuts include a `trigger_id` which an app can use to [open a modal](#creating-modals) that confirms the action the user is taking.
+
+When configuring shortcuts within your app configuration, you'll continue to append `/slack/events` to your request URL.
```

Comment:
```suggestion
When setting up shortcuts within your app configuration, you'll append `/slack/events` to your request URL.
```

Reply:
If we want to convey that it's for all URLs, we could make it something like "Like other URLs, you'll append `slack/events` to your request URL when setting up shortcuts."